EmployeeLib/IPerformanceReviewer.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace EmployeeLib
+{
+    public interface IPerformanceReviewer
+    {
+        public void GeneratePerformanceReview()
+        {
+            // Simulate reviewing a direct report
+            Console.WriteLine("I'm reviewing a direct report's performance.");
```

Comment:
Don't add implementation to an interface, this usually done when you have a library, and want to have backward compatibility which we don't need here